storage/client/src/main/java/com/azure/storage/file/models/FileCopyInfo.java

```diff
@@ -3,5 +3,55 @@
 
 package com.azure.storage.file.models;
 
+import com.azure.core.implementation.util.ImplUtils;
+import java.time.OffsetDateTime;
+
 public final class FileCopyInfo {
+    private String eTag;
+    private OffsetDateTime lastModified;
+    private String copyId;
+    private CopyStatusType copyStatus;
+
+    public FileCopyInfo(final String eTag, final OffsetDateTime lastModified, final String copyId, final CopyStatusType copyStatus) {
```

Comment:
We shouldn't be mixing immutable POJO patterns with fluent patterns. If this class doesn't get used by a consumer of the API other than as a response from the service I think we should remove the fluent portion, if a consumer is expected to create these then lets remove this constructor.

Additionally, if we go the immutable POJO route, I saw that all these values are contained as response headers, might as well just clean up the constructor interface and just pass the headers and grab the values from it.